charts/pxc-db/production-values.yaml

```diff
@@ -207,23 +303,19 @@ proxysql:
 
 logcollector:
   enabled: true
-  image:
-    repository: percona/percona-xtradb-cluster-operator
-    tag: 1.8.0-logcollector
+  image: ""
 
 pmm:
   enabled: false
   image:
     repository: percona/pmm-client
-    tag: 2.12.0
+    tag: 2.18.0
   serverHost: monitoring-service
   serverUser: admin
 
 backup:
   enabled: true
-  image:
-    repository: percona/percona-xtradb-cluster-operator
-    tag: 1.8.0-pxc8.0-backup
+  image: ""
   imagePullSecrets: []
   # - name: private-registry-credentials
   pitr:
```

Comment:
we should add the following password also: `replication: insecure-replication-password` in `secrets.passwords` part.

Reply:
done